### PR TITLE
Include archive versions when generating output

### LINKED ISSUE #31: Html report does not show Old and New versions as well as Semantic Versioning field
Screen shot attached below. For now, I'm forced to upgrade html directly to set versions. As for semantic versioning I'll have to parse html in order to find some (*) or (!) changes and understand  which version should I increment. This functionality was present in Maven plugin.
Am I missing something or it's not implemented yet? And if so, are there any plans to implement it?

na_problem

### PR DESCRIPTION
**Pull request by guillermocalvo**
Resolves #31

### CODE REVIEW

File: src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy [hunk 1]
```diff
@@ -30,7 +30,7 @@ class ClasspathIsUsedFunctionalTest extends BaseFunctionalTest {
                 throw new IllegalStateException("Need to update the byteCode version mapping for Java ${JavaVersion.current()}, you can ref https://javaalmanac.io/bytecode/versions")
         }
         // Superclasses can only be reported if the classpath is present
-        hasTextReport("""
+        hasTextReport("""Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar
```
Comment (Goooler) [inline src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy:33]:
```suggestion
        hasTextReport("""
        Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar
```

Comment (guillermocalvo) [reply src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy:33]: @Goooler There's no `\n` before `Comparing source compatibility` so this change makes the test fail.

Comment (Goooler) [reply src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy:33]: Ooops, and need a `.trim()` in the end.

Comment (guillermocalvo) [reply src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy:33]: If the problem is that this statement is not very readable, we could split it into two different `hasTextReport` checks.

```suggestion
        hasTextReport("Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar")
        hasTextReport("""
`````````

**Review by melix — APPROVED** [review]

**Pull request merged**